woocommerce/woocommerce
# Move the Save draft and Preview buttons to the product editor header

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/37249



### How to test the changes in this Pull Request:




1. Got to `/wp-admin/tools.php?page=woocommerce-admin-test-helper`
2. Under `Features` tab make sure to enable `block-editor-feature-enabled`
3. Then visit `/wp-admin/admin.php?page=wc-admin&path=/add-product`
4. The preview button should open a new tab with the frontend preview
5. The "Save" button should say "Save" for existing products and "Add" for new products (auto-drafts), but should publish the product
6. The "Save draft" should follow the Gutenberg post format, say Save draft if edited and draft status, show Switch to draft if published, show Saved if draft and saved (no edits).



### Other information:




### FOR PR REVIEWER ONLY:
